pytorch_lightning/loops/epoch/evaluation_epoch_loop.py

```diff
@@ -70,6 +70,8 @@ def on_run_start(
         self._dl_max_batches = dl_max_batches
         self._num_dataloaders = num_dataloaders
 
+        self.dataloader_iter = enumerate(dataloader_iter, self.batch_progress.current.ready)
```

Comment:
we don't have the attribute set in init

Reply:
Done !

Reply:
Isn't this redundant with `_prepare_dataloader_iter` in the epoch loop?

remove this in favor of the other?

Reply:
Updated !